webapp/components/user_settings/user_settings_notifications.jsx

```diff
@@ -198,6 +208,11 @@ class NotificationsTab extends React.Component {
         ReactDOM.findDOMNode(this.refs.wrapper).focus();
     }
 
+    handleNotifyCommentsRadio(notifyCommentsLevel) {
+        this.setState({notifyCommentsLevel});
+        ReactDOM.findDOMNode(this.refs.wrapper).focus();
```

Comment:
by getting the reference with `this.refs.wrapper` you don't need `ReactDOM`, so feel free to remove it